app/javascript/pages/Wishlists/Index.tsx

```diff
@@ -168,4 +171,8 @@ const WishlistsPage = ({
   );
 };
 
-export default register({ component: WishlistsPage, propParser: createCast() });
+export default function WishlistsIndex() {
+  const props = cast<Props>(usePage().props);
+
+  return <WishlistsPage {...props} />;
```

Comment:
You could just inline the content of `WishlistsPage` here, it would be less convoluted that way.